### PR TITLE
Asshat

### PR DESCRIPTION
**Pull request by TheNeoGamer42**
## About The Pull Request
Adds Asshat as an alternate job title for assistants
## Why It's Good For The Game
why isnt it?
## Changelog
:cl:
add: Asshat is now a job title.
/:cl:

### CODE REVIEW

**Comment by dee-dubya** [comment]
on one hand, this is hilarious

on the other hand, the last thing we want to encourage is assistants acting like asshats. once you get a few of them, they're like a fucking plague that makes the entire server worse off.

**Comment by TheNeoGamer42** [comment]
should I just make them start the round on arrest then because fuck you if your going to be an asshat

**Comment by TheNeoGamer42** [comment]
actually I dunno how I would do that... hmmmmmmmmmmmm

**Comment by Maldaris** [comment]
Sounds like a greytider senior title

File: code/modules/jobs/job_types/assistant.dm [hunk 1]
```diff
@@ -65,3 +65,7 @@ Assistant
 /datum/outfit/job/assistant/entertainer
 	name = "Assistant (Entertainer)"
 	r_hand = /obj/item/bikehorn //comedy
+
+/datum/outfit/job/assistant/asshat
+	name = "Assistant (Asshat)"
+	backpack_contents = list(/obj/item/toy/gun=1) //terg said it was a good idea at the time, now asshats can shoot up the station with a cap gun
```
Comment (Shadowtail117) [inline code/modules/jobs/job_types/assistant.dm:71]: Requires modularization comment. Comment is unnecessary as well.

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]

**Comment by Shadowtail117** [comment]
Failing unit test:
image

**Comment by Sparrowbird611** [comment]
Counterpoint:

Why is this a good idea.

**Comment by Maldaris** [comment]
> Counterpoint:
> 
> Why is this a good idea. It's not even a real job (I know assistant is barely a job, but it at least sounds like something someone would reasonably have, there's no job in existence literally named asshat).

This was kinda why I suggested it be a seniority title. If you've played Assistant long enough to unlock it, chances are you're an asshat and this does the job of drawing attention to that fact.

**Comment by TheNeoGamer42** [comment]
Well we already have an assistant senior title thats Greytider, and I dunno if margs funny alt title code can handle more than one senior title without exploding or some shit

**Comment by Maldaris** [comment]
> Well we already have an assistant senior title thats Greytider, and I dunno if margs funny alt title code can handle more than one senior title without exploding or some shit

Feels like this would be a more apt senior title, but point taken.

**Comment by Maldaris** [comment]
Also pull master into this branch so the integration tests can pass.

**Comment by SynnGraffkin** [comment]
I think Assistant titles are fine as is.

**Comment by triplezeta** [comment]
yes, agreed

**Comment by MarkSuckerberg** [comment]
Funny, but I don't think it's needed.

**Pull request closed without merging**